src/__tests__/ParseQuery-test.js

```diff
@@ -1289,4 +1289,13 @@ describe('ParseQuery', () => {
       done();
     });
   }));
+
+  it('rewrites User queries when the rewrite is enabled', () => {
```

Comment:
Nit: we actually test two cases not only rewrite is enabled, so maybe change a name?
But really small nit, depends on you.